src/xcode/ENA/ENA/Source/RootCoordinator.swift

```diff
@@ -94,7 +94,13 @@ class RootCoordinator: NSObject, RequiresAppDependencies, UITabBarControllerDele
 
 		return true
 	}
-	
+
+	/// reset increase risk level badge if tab gets selected
+	func tabBarController(_ tabBarController: UITabBarController, didSelect viewController: UIViewController) {
+		guard viewController == homeCoordinator?.rootViewController else { return }
+		homeCoordinator?.resetHomeBadgeRiskStateIncreased()
```

Comment:
There is a `resetBadgeCount` method on the `HomeTableViewModel`, I think we should reset all parts of the badge count in the same place, so either here or in the view model.